app/eventyay/agenda/templates/agenda/widgets/emoji_rating.html

```
@@ -0,0 +1,16 @@
+{% load i18n %}
+<div class="emoji-rating-container">
+  <div class="emoji-rating-group" role="radiogroup" aria-label="{% translate 'Feedback Rating' %}">
+    {% for group, options, index in widget.optgroups %}
+      {% for opt in options %}
+        <div class="emoji-rating-option">
+          <input type="{{ opt.type }}" name="{{ opt.name }}" value="{{ opt.value }}" id="{{ opt.attrs.id }}" class="emoji-rating-radio"{% if opt.attrs.required %} required{% endif %}{% if opt.attrs.checked or opt.selected %} checked{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)